bin/ganga

```diff
@@ -21,17 +21,28 @@
 from __future__ import print_function
 import sys
 
+#First make sure we are using a python version between 2.7 and 3
+if sys.version_info[0] != 2 or sys.version_info[1] < 7:
+    import logging
+    logging.error("Ganga does not support python version %s.%s.%s. Make sure you have a python version 2.7.x in your path" % (sys.version_info[0], sys.version_info[1], sys.version_info[2]))
+    sys.exit(-1)
+
 def standardSetup():
     """Function to perform standard setup for Ganga.
     """   
     import os.path
 
-    # insert the path to Ganga itself - this is required for the installation on CVMFS
+    # insert the path to Ganga itself - this is required for the installations without pip 
     exeDir = os.path.abspath(os.path.normpath(os.path.dirname(os.path.realpath(__file__))))
     gangaDir = os.path.join(os.path.dirname(exeDir), 'ganga' )
-
     sys.path.insert(0, gangaDir)
 
+    #On CVMFS we need to point to the site-packages directory as we don't start the virtualenv
+    if exeDir.startswith('/cvmfs/ganga.cern.ch', 0, 20):
```

Comment:
Could the check of not using virtualenv be made a bit more general? If somebody copied the /cvmfs installation to somewhere else (stupid but they might), then it would stop working which seems a bit silly.

Reply:
I see your point but I am not sure we can, unless we just add this always.

Reply:
Is https://stackoverflow.com/questions/1871549/determine-if-python-is-running-inside-virtualenv helpful?

Reply:
But on CVMFS ganga won't be run in a virtualenv - nor will it be if you copy it. I can imagine users will install ganga via pip both with and without virtualenv so that won't help us.

Reply:
Fair point.